src/Model/ModelManager.php

```diff
@@ -404,23 +404,23 @@ public function addIdentifiersToQuery($class, ProxyQueryInterface $queryProxy, a
     {
         $fieldNames = $this->getIdentifierFieldNames($class);
         $qb = $queryProxy->getQueryBuilder();
+        /** @var ClassMetadata $classMetadata */
```

Comment:
`getMetadata()` has already declared a return type in its `@return` type hint. Why we need this inline docblock?
https://github.com/sonata-project/SonataDoctrineORMAdminBundle/blob/babf4865cc6c4ba8af8ef8b21df8118401566605/src/Model/ModelManager.php#L59-L64
```suggestion
```

Reply:
removed